test/parallel/test-buffer-alloc.js

```diff
@@ -464,6 +464,16 @@ assert.strictEqual(
 // Regression test for https://github.com/nodejs/node/issues/3496.
 assert.strictEqual(Buffer.from('=bad'.repeat(1e4), 'base64').length, 0);
 
+// Regression test for https://github.com/nodejs/node/issues/11987.
+assert.strictEqual(Buffer.from('w0  ', 'base64').length, 1);
+assert.deepStrictEqual(Buffer.from('w0  ', 'base64'),
+                       Buffer.from('w0', 'base64'));
+assert.deepStrictEqual(Buffer.from('w0  ', 'base64'), Buffer.from([195]));
+assert.strictEqual(
+  Buffer.from('3v6YpUFyK0/hitA2tCDIfdYKw0 g ', 'base64').toString('base64'),
+  '3v6YpUFyK0/hitA2tCDIfdYKw0g='
+);
+
```

Comment:
@seishun It’s your PR and I’m not going to tell you what to do. :smile: But really, having slightly more tests than necessary won’t hurt anyone…